.github/CODEOWNERS

```diff
@@ -41,4 +41,8 @@
 /docs/resource/host.md    @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers
 /model/resource/host.yaml @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-system-approvers
 
+# Mobile semantic conventions approvers
```

Comment:
Minor, consider alphabetic ordering.

Reply:
The sections weren't perfectly ordered before (schemas was near the top). I'm happy to do that, but should I move `Schemas` as part of it?

Reply:
I don't have strong opinion so either way should work for me. I would suggest scoping it to mobile semantic conventions only, other parts (that are not perfectly ordered) can be handled in a separate PR.

Reply:
I think the mobile section is already internally sorted alphabetically (`docs` ahead of `mobile`)